docs/Modifying_Clearwater_settings.md

```diff
@@ -10,7 +10,7 @@ Settings in `/etc/clearwater/shared_config` can be safely changed without entire
 
 To change one of these settings, if you are using Clearwater's [automatic configuration sharing](Automatic_Clustering_Config_Sharing.md) functionality:
 
-*   Edit `/etc/clearwater/shared_config` on *one* node and change to the new value.
+*   Edit `/etc/clearwater/shared_config` on *one* node and change to the new value. Changes will be logged to `/var/log/syslog` and to the console.
```

Comment:
This should be in the upload_shared_config bit, not the editing shared config bit, no?